src/api/dependency-discovery/src/github.js

```diff
@@ -0,0 +1,60 @@
+const { request } = require('@octokit/request');
+
+const hasGitHubExpired = (dateTime) => Date.now() - dateTime > 60 * 60 * 1000;
```

Comment:
`datetime` is a `Date` object, so make sure to subtract the same object type here.